.github/workflows/ci.yml

```
@@ -0,0 +1,392 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      # Feature branches beginning with "feature."
+      - feature.*
+    tags:
+      - '**'
+  pull_request:
+
+jobs:
+  sass_spec:
+    runs-on: ubuntu-latest
+
+    name: "sass-spec | Dart ${{ matrix.dart_channel }} | ${{ matrix.async_label }}"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        dart_channel: [stable, dev]
+        async_label: [synchronous]
+        async_args: ['']
+        include:
+          - dart_channel: stable
+            async_label: asynchronous
+            async_args: '--cmd-args --async'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Dart
+        uses: cedx/setup-dart@v2
+        with:
+          release-channel: "${{ matrix.dart_channel }}"
+
+      - name: Install app dependencies
+        run: dart pub get
+
+      # Setup the environment to run the sass-spec runner which is written in Ruby
+      - name: Setup sass-spec
```

```suggestion
      - name: Set up sass-spec
```

Fun grammar fact: "setup" is the noun form of the verb "set up"